crmsh/bootstrap.py

```diff
@@ -1846,6 +1885,9 @@ def update_nodeid(nodeid, node=None):
         else:
             corosync.set_value("totem.nodeid", nodeid)
 
+    atexit.register(os.remove, COROSYNC_CONF_ORIG)
```

Comment:
What this `register` is doing?

Reply:
run `os.remove(COROSYNC_CONF_ORIG)` when the program terminated

Reply:
Ok, thanks!